test/integration/admin_can_view_all_orders_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.